schematic/schemas/data_model_jsonld.py

```diff
@@ -0,0 +1,284 @@
+                    rel_func = self.rel_dict[key]['node_attr_dict']['default']
+
+                # Add appropritae contexts that have been removed in previous steps (for JSONLD) or did not exist to begin with (csv)
+                if key == 'id' and rel_func == get_label_from_display_name:
+                    template[jsonld_key] = 'bts:' + template[jsonld_key]
+                elif key == 'required' and rel_func == convert_bool_to_str:
+                    template[jsonld_key] = 'sms:' + str(template[jsonld_key]).lower()
+        return template
+
+    def clean_template(self, template: dict, data_model_relationships: dict) -> dict:
+        '''Get rid of empty k:v pairs. Fill with a default if specified in the relationships dictionary.
+        Args:
+            template, dict: JSONLD template for a single entry, keys specified in property and class templates.
+            data_model_relationships, dict: dictionary containing information for each relationship type supported.
+        Returns:
+            template: JSONLD template where unfilled entries have been removed, or filled with default depending on specifications in the relationships dictionary.
+        '''
+        for rels in data_model_relationships.values():
+            if rels['jsonld_key'] in template.keys() and not template[rels['jsonld_key']]:
+                if 'jsonld_default' in rels.keys():
+                    template[rels['jsonld_key']] = rels['jsonld_default']
+                else:
+                    del template[rels['jsonld_key']]
+        return template
+
+    def strip_context(self, context_value: str) -> tuple[str]:
+        """Strip contexts from str entry.
+        Args:
+            context_value, str: string from which to strip context from
+        Returns:
+            context, str: the original context
+            v, str: value separated from context
+        """
+        if ':' in context_value:
+            context, v = context_value.split(':')
+        elif '@' in context_value:
+            context, v = context_value.split('@')
+        return context, v
+
+    def reorder_template_entries(self, template:dict) -> dict:
+        '''In JSONLD some classes or property keys have list values. We want to make sure these lists are ordered according to the order supplied by the user.
+        This will look specically in lists and reorder those.
+        Args:
+            template, dict: JSONLD template for a single entry, keys specified in property and class templates.
+        Returns:
+            template, dict: list entries re-ordered to match user supplied order.
+        Note:
+            User order only matters for nodes that are also attributes
+        '''
+        template_label = template['rdfs:label']
+
+        for jsonld_key, entry in template.items():
+            # Make sure dealing with an edge relationship:
+            is_edge = ['True' for k, v in self.rel_dict.items() if v['jsonld_key']==jsonld_key if v['edge_rel'] == True]
```

Comment:
Here, `is_edge` returns either [True] or `[]` right? so does that mean `if if_edge` will always be True? I don't think that's what you intended?